web/pwa/components/history/Month.js

```diff
@@ -11,18 +11,34 @@ export function Month({
   selectedPeriodEnd,
   missionsInPeriod
 }) {
+  const hasWorkMissions = React.useMemo(
+    () => missionsInPeriod.filter(mission => !mission.isHoliday).length > 0,
+    [missionsInPeriod]
+  );
+  const kpis = React.useMemo(() => {
+    let allKpis = renderPeriodKpis(
+      splitByLongBreaksAndComputePeriodStats(
+        activitiesWithNextAndPreviousDay,
+        selectedPeriodStart,
+        selectedPeriodEnd,
+        missionsInPeriod
+      )
+    ).filter(m => m.name !== "service");
+    if (hasWorkMissions) {
+      return allKpis;
+    }
+    return allKpis.filter(m => m.name === "offDays");
```

Comment:
:green_circle: Même remarque ici : 
```suggestion
    ).filter(kpi => kpi.name !== "service");
    if (hasWorkMissions) {
      return allKpis;
    }
    return allKpis.filter(kpi => kpi.name === "offDays");
```